Dockerfile.release

```diff
@@ -1,15 +1,15 @@
-FROM alpine:3.12
-
-ARG BINARY=nri-prometheus
+FROM alpine:3.13
+ARG TARGETOS
+ARG TARGETARCH
```

Comment:
Are we actually using these?

Reply:
we were, it is legacy from the first commit of this PR. 🤣 

I'll get rid of it